### PR TITLE
Add link to make for Windows

### PR DESCRIPTION
**Pull request by andreynering**

### CODE REVIEW

**Comment by bkcsoft** [comment]
not a windows user, what is equation.com?

**Comment by andreynering** [comment]
Os a hard to find a working version of make for Windows, this is why I think it's useful to link one.

This site distribute a working build, I don't know the company personally.

An alternative is use MinGW's one, but it's more work to install.

**Comment by Bwko** [comment]
@andreynering  Maybe you could add 

> An alternative is use MinGW's one

 to  the doc

**Comment by andreynering** [comment]
I think I'll link both.

File: content/doc/installation/from-source.en-us.md [hunk 1]
```diff
@@ -44,7 +44,9 @@ git checkout v1.0.0
 
 ## Build
 
-Since we already bundle all required libraries to build Gitea you can continue with the build process itself. We provide various [make tasks](https://github.com/go-gitea/gitea/blob/master/Makefile) to keep the build process as simple as possible. Depending on your requirements you possibly want to add various build tags, you can choose between these tags:
+Since we already bundle all required libraries to build Gitea you can continue with the build process itself. We provide various [make tasks](https://github.com/go-gitea/gitea/blob/master/Makefile) to keep the build process as simple as possible. (For Windows, you can download make in [this page](http://www.equation.com/servlet/equation.cmd?fa=make)).
```
Comment (tboerger) [inline content/doc/installation/from-source.en-us.md]: Replace `(For Windows, you can download make in [this page](http://www.equation.com/servlet/equation.cmd?fa=make)).` with `If you are using Windows you can download make [here](http://www.equation.com/servlet/equation.cmd?fa=make).`

**Comment by tboerger** [comment]
Maybe you can also mention Chocolatey, the missing package manager for windows: https://chocolatey.org/packages/make

**Comment by andreynering** [comment]
Updated.

Since the text got big, I created a separated page to document Make and the Makefile.

If you guys think it would be better to move that to `Hacking on Gitea` instead, I'm also OK with that.

**Comment by tboerger** [comment]
I'm wouldn't go too much into detail, but that's just my opinion. Drop the make tasks from the bottom and link to the page from the other page

**Comment by andreynering** [comment]
Updated

**Comment by Bwko** [comment]
LGTM

File: content/doc/advanced/make-and-makefile.en-us.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+You can install with your package manager. On Ubuntu, make is included in the `build-essential` package:
+
+```bash
+sudo apt-get install build-essential
+```
+
+### On Windows
+
+If you are using Windows, you can download use one of these distributions of make:
```
Comment (tboerger) [inline content/doc/advanced/make-and-makefile.en-us.md]: Fan Download and use...

Comment (tboerger) [reply content/doc/advanced/make-and-makefile.en-us.md]: Can

File: content/doc/installation/from-source.en-us.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ git checkout v1.0.0
 
 ## Build
 
-Since we already bundle all required libraries to build Gitea you can continue with the build process itself. We provide various [make tasks](https://github.com/go-gitea/gitea/blob/master/Makefile) to keep the build process as simple as possible. Depending on your requirements you possibly want to add various build tags, you can choose between these tags:
+Since we already bundle all required libraries to build Gitea you can continue with the build process itself. We provide various [make tasks](https://github.com/go-gitea/gitea/blob/master/Makefile) to keep the build process as simple as possible. [See here how to get Make](/en-us/make). Depending on your requirements you possibly want to add various build tags, you can choose between these tags:
```
Comment (tboerger) [inline content/doc/installation/from-source.en-us.md]: That is not a Hugo linkref :)

**Comment by andreynering** [comment]
Updated

**Comment by tboerger** [comment]
LGTM

**Pull request merged**